shared/prater/config.yaml

```diff
@@ -3,6 +3,15 @@
 # Extends the mainnet preset
 PRESET_BASE: 'mainnet'
 
+# Transition
+# ---------------------------------------------------------------
+# TBD, 2**256-2**10 is a placeholder
```

Comment:
Fix comment, this is no longer a place holder

Reply:
Good catch, fixed.